it/test/connectors/DeleteInsurancePoliciesConnectorISpec.scala

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-package connectors
+package test.connectors
```

Comment:
can u go through all the files in it/test and remove test
eg package connectors